config/locales/en.yml

```diff
@@ -50,7 +50,10 @@ en:
       url: '/'
       icon: 'logos/logo-gofox.svg'
       class: 'gofox'
-  survey_warning: 
+  survey_warning:
+      title: 'Welcome to the Future of Conservation survey'
```

Comment:
I am thinking I might rename this text to something else but I am not sure what to? @rogeringle

Reply:
It is for the survey warning.